Header header            #header info
float64[] m_coeffs 
float64[] n_coeffs
